pkg/virt-controller/watch/vmi_test.go

```diff
@@ -3175,76 +3162,80 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 				expectPodStatusUpdateFailed(pod)
 			})
 
-			It("cannot handle dynamic network attachment when adding an interface", func() {
-				fakeHotPlugRequest(vmi, []AddInterfaceOptions{{
-					NetworkAttachmentDefinitionName: "net1",
-					Name:                            "iface1",
-				}})
-				Expect(controller.handleDynamicInterfaceRequests(
+			It("pod multus status cannot be updated", func() {
+				Expect(controller.updateMultusAnnotation(
 					vmi.Namespace, vmi.Spec.Domain.Devices.Interfaces, vmi.Spec.Networks, pod)).To(HaveOccurred())
 			})
 		})
 
-		Context("hotplug operation", func() {
+		Context("pod status update", func() {
+			const mac1 = "abc"
```

Comment:
Why the mac addresses have been added?

Reply:
Not really sure how it is related to the code changes done here, but I will not dig into it further.